docs/edit/versions.md

```diff
@@ -0,0 +1,27 @@
+System-tests uses [SemVer](https://semver.org/), where a version can be seen as : `1.2.3-prerelease+build`.
+
+* `1.2.3` is the trio `major`, `minor` and `patch`
+* `prerelease` is an optional label, usually something like `rc1`, `dev` ...
+* `build` is also an optional label, usually a commit SHA.
+
+And a very important point, in SemVer `1.2.3` < `1.2.4-rc1` < `1.2.4`.
```

Comment:
is it ? 

https://semver.org/#spec-item-11 says : 

> When major, minor, and patch are equal, a pre-release version has lower precedence than a normal version:
>
> Example: 1.0.0-alpha < 1.0.0.

Reply:
Ok nevermind you're right, it's specified in the 2.0 SemVer spec (wasn't super clear in 1.0). I got confused (again) because in node, npm does it differently.
I've added some test-the-tests for clarification in this PR, just to make sure all our use cases here are correct and expected: https://github.com/DataDog/system-tests/pull/3281